parlai/core/torch_agent.py

```diff
@@ -468,6 +481,13 @@ def add_cmdline_args(cls, argparser):
             help='Epsilon value for Adam optimizers. Set to 1e-6 if your '
             'large model has stability issues, but prefer the default.',
         )
+        optim_group.add_argument(
+            '--adafactor-eps',
+            default='1e-30,1e-3',
```

Comment:
are these recommended values? i.e. might be good to have them marked as `recommended=` in the `add_argument` function